cmd/backup-manager/app/util/util.go

```diff
@@ -135,6 +135,37 @@ func ConstructBRGlobalOptionsForBackup(backup *v1alpha1.Backup) ([]string, strin
 	return args, remotePath, nil
 }
 
+// ConstructMydumperOptionsForBackup constructs mydumper options for backup
+func ConstructMydumperOptionsForBackup(backup *v1alpha1.Backup) []string {
+	var args []string
+	defaultOptions := []string{
+		"--long-query-guard=3600",
+		"--tidb-force-priority=LOW_PRIORITY",
+		"--verbose=3",
+		"--compress-protocol",
+		"--threads=16",
+		"--rows=10000",
+		"--skip-tz-utc",
+	}
+	defaultTableRegexOptions := []string{
+		"--regex",
+		"^(?!(mysql|test|INFORMATION_SCHEMA|PERFORMANCE_SCHEMA|METRICS_SCHEMA|INSPECTION_SCHEMA))",
+	}
+	config := backup.Spec.Mydumper
+	if config == nil {
+		args = append(args, defaultOptions...)
+		args = append(args, defaultTableRegexOptions...)
+		return args
+	}
+	if len(config.Options) != 0 {
+		args = append(args, config.Options...)
+	}
+	if config.TableRegex != nil {
```

Comment:
If `config.TableRegex` is nil, we need to add the default regex config.
If `config.Options` is nil, we need to add the default options config.

Reply:
Please add unit test for this function.